code/modules/jobs/job_types/roguetown/courtier/jester.dm

```diff
@@ -20,7 +20,7 @@
 	min_pq = -4 //silly jesters are funny so low PQ requirement
 	max_pq = null
 	round_contrib_points = 2
-	social_rank = SOCIAL_RANK_NOBLE
+	social_rank = SOCIAL_RANK_YEOMAN
```

Comment:
Court jesters were privledged positions in history. The fact that people are playing them as 'silly little guys' without taking into account the idea of Jesters Privledge is a bit of a shame.
That said- I'd say they should be atleast minor noble, social status wise.